templates/docker.j2.service

```diff
@@ -8,7 +8,7 @@ Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/docker daemon -H unix:///var/run/docker.sock
+ExecStart=/usr/bin/dockerd -H unix:///var/run/docker.sock
```

Comment:
Was this change necessary? This has also come in issue #22 
If this fixes that, let's also close that one.

Reply:
It is needed as the old way is deprecated. This fixes #22 as well